main.py

```diff
@@ -35,7 +38,9 @@ async def change_status() -> None:
 async def on_ready() -> None:
     print("Bot is Ready.")
     print(f"Logged in as: {client.user.name} : {client.user.id}")
-
+    if client.first_on_ready:
+        client.first_on_ready = False
+        client.pool = await asyncpg.create_pool(database="@Jana", host="127.0.0.1", min_size=20, max_size=100) #  @Jana : link to the connection credentials
```

Comment:
The credentials should be defined in `.env` file, you can use `os` module to get the environmental variable.

Reply:
I wrote them as @Jana, because I leave that part to him. That is currently only a placeholder. He'll probably make that a .env, yeah